edge-proxy/src/run.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
```

Comment:
I don't know about kubernetes, @avranju would you please help to review it?

Reply:
Done. Thanks.